fluency-fluentd/src/main/java/org/komamitsu/fluency/fluentd/FluencyBuilderForFluentd.java

```diff
@@ -97,6 +99,10 @@ public void setSslEnabled(boolean sslEnabled)
         this.sslEnabled = sslEnabled;
     }
 
+    public void setSslSocketFactory(SSLSocketFactory sslSocketFactory) {
```

Comment:
Ah, sorry. One more.

Reply:
All set! Sorry about that -- my IDE collapses them into one line + autogens so I don't see this unless I click the little `+` sym 😅